### PR TITLE
[4.0] Implement the workflow into the multilang sample data

### PR DESCRIPTION
**Pull request by bembelimen**
Pull Request for Issue #21707 . (second part)

### Summary of Changes
This PR let the multilang sample data creating a new Workflow to ensure, that the articles are published

### Testing Instructions
- Install a 2nd language
- Apply the "Multilingual Sample Data" (from the dashboard)
- check the workflows, a new workflow called "Sample Workflow (xx-xx)" is created and assigned to the specific new sample data category (xx-xx is the language tag).

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_sampledata_multilang.ini [hunk 1]
```diff
@@ -16,6 +18,7 @@ PLG_SAMPLEDATA_MULTILANG_ERROR_MAINMENU_MODULE="Step %1$u: Failed unpublishing t
 PLG_SAMPLEDATA_MULTILANG_ERROR_MENUMODULES="Step %1$u: Failed creating the %2$s menu module."
 PLG_SAMPLEDATA_MULTILANG_ERROR_MENUS="Step %1$u: Failed creating the Main menu (%2$s)."
 PLG_SAMPLEDATA_MULTILANG_ERROR_SWITCHER="Step %1$u: Failed creating and enabling the %2$s."
+PLG_SAMPLEDATA_MULTILANG_ERROR_WORKFLOW="Step %1$u: Failed creating the Workflow"
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_sampledata_multilang.ini]: please add a full stop at the end

File: administrator/language/en-GB/en-GB.plg_sampledata_multilang.ini [hunk 2]
```diff
@@ -4,6 +4,8 @@
 ; Note : All ini files need to be saved as UTF-8
 
 PLG_SAMPLEDATA_MULTILANG="Sample Data - Multilingual"
+PLG_SAMPLEDATA_MULTILANG_CONTENT_WORKFLOW_DESCRIPTION="A sample workflow for \"Sample Data - Multilingual\""
```
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_sampledata_multilang.ini:7]: this string is not needed and can be removed

Comment (bembelimen) [reply administrator/language/en-GB/en-GB.plg_sampledata_multilang.ini:7]: it's used here: https://github.com/joomla/joomla-cms/pull/22123/files#diff-1474adb453300a3f597fa71ca93af729R997

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on aa774741056b367ab8461eb3f11c8ed76764d745<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/22123">issues.joomla.org/tracker/joomla-cms/22123</a>.</sub>

**Pull request merged**